src/components/Network.js

```diff
@@ -33,27 +34,45 @@ class Network extends React.Component {
       connected_peers: [],
       known_peers: [],
       loaded: false,
-      isLoading: false
+      isLoading: false,
+      peers: [],
+      peer_id: '',
     }
 
     this.loadData = this.loadData.bind(this);
   }
 
   componentDidMount() {
-    this.loadData();
-    this.loadTimer = setInterval(this.loadData, 1000);
+    const { match: { params } } = this.props;
+
+    networkApi.getPeers().then((peers) => {
```

Comment:
What if this request fails? What happens? Shouldn't we show an error message and keep trying every N seconds?

Reply:
Ok. Let's create another issue to add an error screen, please.

Reply:
@msbrogli created #85